frontend/src/scenes/insights/insightLogic.test.ts

```diff
@@ -1276,4 +1300,43 @@ describe('insightLogic', () => {
             ])
         })
     })
+
+    describe('saving query based insights', () => {
+        beforeEach(async () => {
+            logic = insightLogic({
+                dashboardItemId: 'new',
+            })
+            logic.mount()
+        })
+
+        it('sends query when saving', async () => {
+            jest.spyOn(api, 'create')
+
+            await expectLogic(logic, () => {
+                logic.actions.setInsight(
+                    { filters: {}, query: { kind: NodeKind.DataTableNode } as DataTableNode },
+                    { overrideFilter: true }
+                )
+                logic.actions.saveInsight()
+            })
+
+            const mockCreateCalls = (api.create as jest.Mock).mock.calls
+            // there will be at least two used client query ids
+            // the most recent has not been cancelled
+            // the one before that has been
```

Comment:
I think this comment is copypasta from somewhere else? Or else I don't understand it :D